pandas/core/arrays/sparse.py

```diff
@@ -1650,13 +1651,11 @@ def sparse_unary_method(self):
 
     @classmethod
     def _create_arithmetic_method(cls, op):
+
+        @ops.unpack_and_defer
```

Comment:
Both here and below this is technically a change since in the status quo this doesn't defer to DataFrame.  Also doesn't call item_from_zerodim ATM.